scripts/build_imagemagick

```diff
@@ -13,56 +13,23 @@ function handle_error() {
 }
 trap handle_error EXIT SIGINT
 
-function add_xenial_repositories() {
-  cat <<-EOT >> /etc/apt/sources.list
-  deb http://archive.ubuntu.com/ubuntu/ xenial main universe
-  deb http://security.ubuntu.com/ubuntu xenial-security main
-EOT
-}
+# avoid error: E: You must put some 'deb-src' URIs in your sources.list
+echo "deb-src http://archive.ubuntu.com/ubuntu noble main universe" >> /etc/apt/sources.list
 
-# add source packages repository to sources.list
-function add_source_repositories() {
-  echo "deb-src http://archive.ubuntu.com/ubuntu/ xenial main universe" >> /etc/apt/sources.list
-  if [[ $STACK == "heroku-20" ]]; then
-    add_xenial_repositories
-  fi
-}
 
-add_source_repositories
 
 echo "⚙️ Getting libraries and dependencies..."
 apt-get update -y
 apt-get install -y build-essential autoconf libtool git-core
-if [[ $STACK == "heroku-20" ]]; then
-  echo "⚙️ Getting additional dependencies for the selected stack..."
-  apt-get install -y libomp5 libomp-10-dev libjasper-dev
```

Comment:
See [comment about libjasper-dev.](https://github.com/retailzipline/heroku-buildpack-imagemagick-heif/pull/1/files#r1747416395).

libomp5 libomp-10-dev are for OpenMP support. Ubuntu 24.04 uses `gcc 13.2.0` which has support for OpenMP built-in (see https://gcc.gnu.org/wiki/openmp. and https://www.openmp.org/resources/openmp-compilers-tools/).

Reply:
```bash
$ apt show libomp-dev
Package: libomp-dev
Version: 1:18.0-59~exp2
Priority: extra
Section: universe/libdevel
Source: llvm-defaults (0.59~exp2)
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: LLVM Packaging Team <pkg-llvm-team@lists.alioth.debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 11.3 kB
Depends: libomp-18-dev (>= 18~)
Download-Size: 5366 B
APT-Sources: http://archive.ubuntu.com/ubuntu noble/universe amd64 Packages
Description: LLVM OpenMP runtime - dev package

$ apt show libomp5
Package: libomp5
Version: 1:18.0-59~exp2
Priority: extra
Section: universe/libs
Source: llvm-defaults (0.59~exp2)
Origin: Ubuntu
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Original-Maintainer: LLVM Packaging Team <pkg-llvm-team@lists.alioth.debian.org>
Bugs: https://bugs.launchpad.net/ubuntu/+filebug
Installed-Size: 11.3 kB
Depends: libomp5-18 (>= 18~)
Download-Size: 5348 B
APT-Sources: http://archive.ubuntu.com/ubuntu noble/universe amd64 Packages
Description: LLVM OpenMP runtime
```